js/form-validation.js

```diff
@@ -0,0 +1,23 @@
+// Cliente side validation
+
+// Get elements for the form, need the actual form, the email
+// and the div id to show the error.
```

Comment:
[OPTIONAL] Avoid leaving pieces of comments here, comments that simply repeat what the code is already saying can be redundant and a waste of space. 😉